src/components/Typography.ts

```diff
@@ -0,0 +1,68 @@
+import { css } from 'styled-components';
+
+export const Typography = {
+  H1: css`
+    font-family: 'Helvetica Neue', sans-serif;
+    font-size: 24px; 
+    font-weight: 300;
+    color: #000000;
+
+    @media (min-width: 768px) {
+      font-size: 32px; 
+    }
+  `,
```

Comment:
lets make a breakpoint object with key value pairs of sizes and their values.
for example: 
`
const breakpoints = {
sm: 500,
m: 768,
lg: 1200
}
`
this way if you'll have to adjust or change a break point, you'll do it in one place only.

Reply:
Thanks,
I have created the ScreenBreakPoints object as you suggested. This object contains key value pairs for different breakpoints.